node/src/Pos/Wallet/Web/ClientTypes/Instances.hs

```diff
@@ -6,52 +6,40 @@ module Pos.Wallet.Web.ClientTypes.Instances () where
 
 import           Universum
 
-import qualified Data.ByteArray                       as ByteArray
-import qualified Data.ByteString                      as BS
-import           Data.List                            (intersperse, partition)
-import           Data.Text                            (splitOn)
+import qualified Data.ByteArray as ByteArray
+import qualified Data.ByteString as BS
+import           Data.List (intersperse, partition)
+import           Data.Text (splitOn)
```

Comment:
I think the "problem" is that `cardano-sl-1.0` uses the old `stylish-haskell` configuration. @gromakovsky how do we care about formatting for this feature branch? Do we need to keep consistency? My 2 cents is that we should keep the *new* indentation strategy, so that when we cherry pick this one from `cardano-sl-1.0` into `master`, conflicts will be less severe. Thoughts?

Reply:
@adinapoli-iohk I was using new `stylish-haskell` from `master` (copied it over to ~/.stylish-haskell)

Reply:
> My 2 cents is that we should keep the new indentation strategy, so that when we cherry pick this one from cardano-sl-1.0 into master, conflicts will be less severe. Thoughts?

Makes sense, I agree, but I guess the last word is up to @dcoutts because all PRs to `cardano-sl-1.0` should pass his review anyway.

Reply:
Ok, then I will leave it as it is (as this is styled with `stylish-haskell` from `master`)